src/taoensso/faraday.clj

```diff
@@ -369,15 +411,18 @@
 
   TableDescription
   (as-map [d]
-    {:name          (keyword (.getTableName d))
-     :creation-date (.getCreationDateTime d)
-     :item-count    (.getItemCount d)
-     :size          (.getTableSizeBytes d)
-     :throughput    (as-map (.getProvisionedThroughput  d))
-     :indexes       (as-map (.getLocalSecondaryIndexes  d)) ; DEPRECATED
-     :lsindexes     (as-map (.getLocalSecondaryIndexes  d))
-     :gsindexes     (as-map (.getGlobalSecondaryIndexes d))
-     :status        (utils/un-enum (.getTableStatus d))
+    {:name                 (keyword (.getTableName d))
+     :creation-date        (.getCreationDateTime d)
+     :item-count           (.getItemCount d)
+     :size                 (.getTableSizeBytes d)
+     :throughput           (as-map (.getProvisionedThroughput d))
+     :indexes              (as-map (.getLocalSecondaryIndexes d)) ; DEPRECATED
+     :lsindexes            (as-map (.getLocalSecondaryIndexes d))
+     :gsindexes            (as-map (.getGlobalSecondaryIndexes d))
+     :stream-specification (as-map (.getStreamSpecification d))
```

Comment:
The AWS SDKs can be incredibly verbose. My general preference is to use the shortest name that's unambiguous and intelligible.

I'll note that the rest of Faraday uses this convention, so the suggested change would also help to maintain consistency with the rest of the API.